assets/source/setup-guide/app/components/SaveSettingsButton.test.js

```diff
@@ -0,0 +1,42 @@
+jest.mock( '../helpers/effects', () => {
+	return {
+		useCreateNotice: () => () => {},
+		useSettingsSelect: () => {},
+		useSettingsDispatch: () => () => {},
+	};
+} );
+jest.mock( '@woocommerce/tracks', () => {
+	return {
+		recordEvent: jest.fn(),
+	};
+} );
```

Comment:
:nail_care: 
If you are mocking default export explicitly, instead of auto mocking everything, WDYT of giving this mock a name.
To get 
>     expect(jest.fn()).toBeCalledWith(...expected)

instead of 
>     expect(jest.fn()).toBeCalledWith(...expected)


```suggestion
jest.mock( '@woocommerce/tracks', () => {
	return {
		recordEvent: jest.fn().mockName( 'recordEvent' ),
	};
} );
```


Otherwise, I'd just do jest.mock( '@woocommerce/tracks' );

Reply:
Fixed in https://github.com/woocommerce/pinterest-for-woocommerce/pull/295/commits/0afbde0966e4b9124d2ae8a573d788ea88213746